WordPress/Classes/ViewRelated/Post/Geolocation/PostGeolocationViewController.m

```diff
@@ -135,37 +119,66 @@ - (void)updateLocation
     [self refreshView];
 }
 
+- (void)showLocationPermissionDisabled {
+    [self showLocationError:[NSError errorWithDomain:kCLErrorDomain code:kCLErrorDenied userInfo:nil]];
+}
+
+- (void)showLocationError:(NSError *)error {
+    NSString *title = NSLocalizedString(@"Location", @"Title for alert when a generic error happened when trying to find the location of the device");
```

Comment:
Looks like `PostSettingsViewController` does pretty much the same thing.  Maybe this could be factored out into a helper method (maybe on `LocationService`)?